docs/run-tests-locally.md

```diff
@@ -37,59 +26,26 @@ Accessibility and UI tests are written using [Cypress](https://www.cypress.io/),
-
     ```
 
-If you need to update the mocked test data for UI and Accessibility tests see [update test data](./update-test-data.md)
-
-### Security testing with ZAP
-
-The Cypress tests can also be run, proxied via [OWASP ZAP](https://zaproxy.org) for passive security scanning of the application.
-
-These can be run using the configured `docker-compose.yml`, which will spin up containers for the ZAP daemon and the Cypress tests, including all networking required. You will need to update any config in the file before running
-
-Create a `.env` file for docker, this file needs to include
-
-- all of your required cypress configuration
-- HTTP_PROXY e.g. `http://zap:8080`
-- ZAP_API_KEY, can be any random guid
-
-Example env:
-
-```env
-URL=<Enter URL>
-USERNAME=<Enter username>
-API=<Enter API>
-API_KEY=<Enter API key>
-AUTH_KEY=<Enter auth key>
-HTTP_PROXY=http://zap:8080
-ZAP_API_KEY=<Enter random guid>
+2. DFE machines may have trouble running Cypress for the first time or installing dependancies due to internal proxies and auth. If in the instance this looks like its occuring on setup you will need to run the following command to bypass this for the installation.
 
-```
-
-**Note**: You might have trouble running this locally because of docker thinking localhost is the container and not your machine
-
-To run docker compose use:
-
-`docker-compose -f docker-compose.yml --exit-code-from cypress`
-
-**Note**: `--exit-code-from cypress` tells the container to quit when cypress finishes
+    ```bash
+    export NODE_TLS_REJECT_UNAUTHORIZED=0'
+    ```
 
-You can also exclude URLs from being intercepted by using the NO_PROXY setting
+3. Open a terminal in your repository and run:
 
-e.g. NO_PROXY=google.com,yahoo.co.uk
+    ```bash
+    cd tests/DFE.FindInformationAcademiesTrusts.CypressTests
 
-Alternatively, you can run the Cypress tests against an existing ZAP proxy by setting the environment configuration
+    # run tests 
+    npx cypress open --> #This will run Cypress for the first time and notify you of such, it should then setup Cypress locally for you. 
+    ```
 
-```env
-HTTP_PROXY="<zap-daemon-url>"
-NO_PROXY="<list-of-urls-to-ignore>"
-```
+4. Once the above step is ran you should have the Cypress UI open showing your test specs - to run these you click on the test spec and it should then run all the tests within said spec.
 
-and setting the runtime variables
+    - If the test passes you should see a green tick next to said test with confirmation that all tests within the spec are passing if multiple.
+    - If the test fails it should show you within the runner what the failed test step is with a screenshot of where it failed to help you in debugging the fail. (N.b Cypress will rerun the open spec everytime you save one of your open files)
 
-`zapReport=true,zapApiKey=<zap-api-key>,zapUrl="<zap-daemon-url>"`
+### Security testing with ZAP - Currently deprecated and will be reintroduced down the line
```

Comment:
To keep docs tidy, we can should delete this line and reintroduce it when ZAP is working again